rwengine/src/loaders/LoaderGXT.cpp

```diff
@@ -8,20 +8,21 @@
 #include <platform/FileHandle.hpp>
 
 #include "data/GameTexts.hpp"
+#include "rw/bit_cast.cpp"
 
 void LoaderGXT::load(GameTexts &texts, FileHandle &file) {
     auto data = file->data;
 
     data += 4;  // TKEY
 
-    std::uint32_t blocksize = *(std::uint32_t *)data;
+    std::uint32_t blocksize = bit_cast<std::uint32_t>(*data);
 
     data += 4;
 
     auto tdata = data + blocksize + 8;
 
     for (size_t t = 0; t < blocksize / 12; ++t) {
-        size_t offset = *(std::uint32_t *)(data + (t * 12 + 0));
+        size_t offset = bit_cast<std::uint32_t>(data + (t * 12 + 0));
```

Comment:
Aren't semantics changed here? I'm not familiar with the `bit_cast` function, but above you dereference data and pass that to the `bit_cast`. Here, you just calculate the address, don't dereference it and pass it to `bit_cast`

Reply:
You're right, I'll rebase.

(Casting ptr and dereferencing can cause UB, if padding is different. bit_cast handles it)